package.json

```diff
@@ -10,7 +10,7 @@
     "http-server": "^14.1.0",
     "jwt-decode": "^3.1.2",
     "react": "^17.0.2",
-    "react-bootstrap": "^2.1.1",
+    "react-bootstrap": "^2.1.2",
```

Comment:
is this still needed in here 👀

Reply:
Will get rid of it once we get stop using bootstrap in the components